### PR TITLE
Update identity-access-policies-guest-access.md

### LINKED ISSUE #11601: Incorrect configuration guidance for guest/external users CA policy
The directions in the table to Require MFA always for guests and external users is incorrect:

Create this new policy and configure:
For Assignments > Users and groups > Include, choose Select users and groups, and then select All guest and external users.
For Assignments > Conditions > Sign-in, leave all options unchecked to always enforce multi-factor authentication (MFA).

First, there is no Conditions > Sign-in, it should be Conditions > Sign-in risk.
Second, I believe we need to select all risk levels.

As you can see below, you can't save if we leave all options unchecked ;)

image

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0a4806dd-a0d0-966c-8130-e752676090a5
* Version Independent ID: 6403c614-26bc-f037-262a-1ec33fc7d385
* Content: [Identity and device access policies for allowing guest and external user B2B access - Microsoft 365 for enterprise - Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/identity-access-policies-guest-access?view=o365-worldwide#updating-the-common-policies-to-allow-and-protect-guests-and-external-user-access)
* Content Source: [microsoft-365/security/office-365-security/identity-access-policies-guest-access.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/identity-access-policies-guest-access.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11601

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by nathanmcnulty** [comment]
Thanks for the update @msbemba , but I'm not sure the commit achieves the desired outcome.

`Sign-in risk** and select the Sign-in risk level this policy will apply to leave all options unchecked to enforce multi-factor authentication (MFA).`

We cannot save the policy unless we check at least one option, and based on the desired outcome of the doc, I believe we would need to check all options (High, Medium, Low, and None).

Thanks!

Nathan

File: microsoft-365/security/office-365-security/identity-access-policies-guest-access.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The following table lists the policies you either need to create and update. The
 
 |Protection level|Policies|More information|
 |---|---|---|
-|**Starting point**|[Require MFA always for guests and external users](identity-access-policies.md#require-mfa-based-on-sign-in-risk)|Create this new policy and configure: <ul><li>For **Assignments > Users and groups > Include**, choose **Select users and groups**, and then select **All guest and external users**.</li><li>For **Assignments > Conditions > Sign-in**, leave all options unchecked to always enforce multi-factor authentication (MFA).</li></ul>|
+|**Starting point**|[Require MFA always for guests and external users](identity-access-policies.md#require-mfa-based-on-sign-in-risk)|Create this new policy and configure: <ul><li>For **Assignments > Users and groups > Include**, choose **Select users and groups**, and then select **All guest and external users**.</li><li>For **Assignments > Conditions > Sign-in risk** and select the Sign-in risk level this policy will apply to leave all options unchecked to enforce multi-factor authentication (MFA).</li></ul>|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/identity-access-policies-guest-access.md]:
```suggestion
|**Starting point**|[Require MFA always for guests and external users](identity-access-policies.md#require-mfa-based-on-sign-in-risk)|Create this new policy and configure: <ul><li>For **Assignments > Users and groups > Include**, choose **Select users and groups**, and then select **All guest and external users**.</li><li>For **Assignments > Conditions > Sign-in risk** and select the Sign-in risk level this policy will apply to. Leave all options unchecked to enforce multi-factor authentication (MFA).</li></ul>|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder

**Comment by v-ratulach** [comment]
Merged. Thanks!

**Pull request merged**